scalableminds/webknossos
# Added loading spinner when requesting tasks

See https://discuss.webknossos.org/t/get-a-new-task-you-think-no-tasks-because-they-are-beeing-loaded/308

### Mailable description of changes (needs to be understandable by webknossos mailing list people):
- show a loading spinner when requesting a new task

### Steps to test:
- Go to dashboard --> tasks
- see a spinner when navigating to "tasks" tab
- see a spinner when requesting new tasks

------
